openaps/devices/add.py

```diff
@@ -25,4 +25,6 @@ def main (args, app):
   # app.config.write(sys.stdout)
   app.config.save( )
   print "added", device.format_url( )
+  repo = app.git_repo( )
+  repo.git.add(['openaps.ini'], write_extension_data=False)
```

Comment:
I think this fixes https://github.com/openaps/openaps/commit/ca313055fed1493c88d52a1b093661c3a1430b1f